content/statsServiceMap.md

```diff
@@ -1,16 +1,13 @@
 ---
 author: David Watkins
-date: 2016-06-23
+date: 2016-07-29
 slug: stats-service-map
-draft: True
+status: draft
 title: Using the dataRetrieval Stats Service
-categories: Data Science
-tags: 
-  - R
-  - dataRetrieval
-image: static/stats-service-map/plot-1.png
+categories:
+  - r
```

Comment:
Try using:
devtools::install_github("USGS-R/USGSmarkdowntemplates")
Then you should be getting our more tailored header (with the right categories and "R" tag